ansible/configs/kni-osp/pre_software.yml

```diff
@@ -0,0 +1,51 @@
+---
+
+- name: Step 003 - Configure all hosts with Repositories, Common Files and Set environment key
+  hosts:
+  - utility
```

Comment:
I'm a bit confused with this. This step should be targeting all hosts, but I understand you wanted to limit it. 

You were going to merge the provision and bastion role, but I'm unsure of what "utility" means.

Reply:
utility used to be the provision host role, which I added the bastion role to to avoid having to make changes everywhere